Doc/c-api/datetime.rst

```diff
@@ -28,7 +71,7 @@ Type-check macros:
 .. c:function:: int PyDate_Check(PyObject *ob)
 
    Return true if *ob* is of type :c:data:`PyDateTime_DateType` or a subtype of
-   :c:data:`PyDateTime_DateType`.  *ob* must not be ``NULL``.  This function always
+   :c:data:`!PyDateTime_DateType`.  *ob* must not be ``NULL``.  This function always
```

Comment:
Remark: You could keep the two links to format them the same way. Same remark below.

Reply:
They are formatted the same way, but only the first one creates a link. I do not know where it is specified, but the style guide is that you should avoid repeating the same link in one paragraph (or even in few short paragraphs).